docs/server/source/server-reference/configuration.md

```diff
@@ -248,6 +251,38 @@ export BIGCHAINDB_WSSERVER_PORT=9985
 }
 ```
 
+## wsserver.advertised_scheme, wsserver.advertised_host and wsserver.advertised_port
+
+These settings are for the advertising the Websocket URL to external clients in
+the root API endpoint. These configurations might be useful if your deployment
+is hosted behind a firewall, NAT, etc. where the exposed public IP or domain is
+different from where BigchainDB is running.
+
+**Example using environment variables**
+```text
+export BIGCHAINDB_WSSERVER_ADVERTISED_SCHEME=wss
+export BIGCHAINDB_WSSERVER_ADVERTISED_HOST=mybigchaindb.com
+export BIGCHAINDB_WSSERVER_ADVERTISED_PORT=443
+```
+
+**Example config file snippet**
+```js
+"wsserver": {
+    "advertised_scheme": "wss",
+    "advertised_host": "mybigchaindb.com",
+    "advertised_port": 443
```

Comment:
@TimDaub I kept it at the same level as it isn't something like:
```
"wsserver": {
    "internal" : {
        "port": ...
    },
    "advertised": {
        "port": ...
    }
}
```

but goes hand in hand with what is specified right next to each other, or at the same indent level.